contribs/github-bot/requirements/branch.go

```diff
@@ -0,0 +1,54 @@
+package requirements
+
+import (
+	"fmt"
+
+	"github.com/gnolang/gno/contribs/github-bot/client"
+	"github.com/gnolang/gno/contribs/github-bot/utils"
+
+	"github.com/google/go-github/v64/github"
+	"github.com/xlab/treeprint"
+)
+
+// Pass this to UpToDateWith constructor to check the PR head branch
+// against its base branch
+const PR_BASE = "PR_BASE"
```

Comment:
Why is this exported?

Reply:
To be used from the config file if you want this `UpToDateWith` requirement to be checked against the PR base branch (and not a custom / user specified base branch).